Converter/ConverterRegistry.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/*
+ * This file is part of the Integrated package.
+ *
+ * (c) e-Active B.V. <integrated@e-active.nl>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Integrated\Bundle\ExportBundle\Converter;
+
+use Integrated\Common\ContentType\ContentTypeInterface;
+
+/**
+ * @author Jeroen van Leeuwen <jeroen@e-active.nl>
+ */
+class ConverterRegistry
+{
+    /**
+     * @var ConverterInterface[]
+     */
+    protected $converters = [];
+
+    /**
+     * @param ContentTypeInterface|null $contentType
+     * @return ConverterInterface[]
+     */
+    public function getConverters(ContentTypeInterface $contentType = null)
+    {
+        $converters = [];
+        if (null !== $contentType) {
+            foreach ($this->converters as $converter) {
+                if ($converter->supports($contentType)) {
+                    $converters[] = $converter;
+                }
+            }
+        } else {
+            $converters = $this->converters;
+        }
+
+        return $converters;
+    }
+
+    /**
+     * @param ConverterInterface[] $converters
+     * @return $this
+     */
+    public function setConverters($converters)
```

Comment:
A set implies that the old converters are overriden but this method does a add instead.